### DISCUSSION TITLE
how does unleash api supported different language which library used in front-end and backend

### DISCUSSION THREAD

**Discussion by gsakkar** [Unleash/unleash#6978] [General]
{
    "id": "06159de5-bec2-4358-8e39-18950265427a",
    "name": "userWithId",
    "constraints": [],
    "parameters": {
        "userIds": "ゆうたろう@メール.にほん"
    }
}

**Comment by ivarconr** [comment]
I am not sure if I fully understand your question. If it is whether Unleash will support different "characters" you should be safe as long as you you stick to `utf-8`.
